web/public/gameData_v1.0-23.json

```diff
@@ -10221,7 +10633,377 @@
                 "power": 4
             },
             "isAlternate": false,
-            "isFicsmas": false
+            "isFicsmas": false,
+            "isPowerGenerator": false
+        }
+    ],
+    "powerGenerationRecipes": [
+        {
+            "id": "GeneratorBiomass_Automated_GenericBiomass",
+            "displayName": "Biomass Burner (Biomass)",
+            "ingredients": [
+                {
+                    "part": "GenericBiomass",
+                    "amount": 0.16666666666666666,
+                    "perMin": 10
+                }
+            ],
+            "products": [],
+            "building": {
+                "name": "BiomassBurner",
+                "power": 30
+            },
+            "isAlternate": false,
+            "isFicsmas": false,
```

Comment:
I guess this is in keeping with the Recipe interface, but I believe these are pointless.